nss-ivory-trolls/WorkforceManagement
# Computers assign employees

# Description

**Given** a user wants to assign a new computer to an employee
**When** the user clicks on the create computer affordance
**Then** the creation form should include an affordance that lists all employees

**Given** the user is viewing the computer creation for
**When** the user fills out all properties for the computer
**And** has chosen an employee
**Then** the user should be redirected back to the list of computers

**Given** the user has created a new computer
**When** the user is redirected to the computer list
**Then** the currently assigned employee's full name should be displayed in the list


## Type of change


# Testing Instructions for Change Made

```
git fetch --all
git checkout computers-assign-employees
```

1. Run the application
2. Click on Computers and confirm that Employee Name section appears in list of computers with the assigned employee's full name
3. Click Create New
4. Fill out form and assign employee
5. Confirm new computer is in list with employee assigned
6. Create another computer without assigning an employee
7. Confirm computer is in list without employee assigned


# Checklist:
